.github/workflows/release-dev.yaml

```diff
@@ -81,8 +148,8 @@ jobs:
       - name: Get Intermediate Artifacts
         uses: actions/download-artifact@master
         with:
-          name: bin-artifacts
-          path: dist
+          name: testkube_windows
+          path: windows
```

Comment:
we should probably also update getting the artifacts for brew and linux, they are still using dist